carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/ownership/Ownership.java

```diff
@@ -68,15 +71,15 @@ public static String getMethodOwner(ITestResult result, OwnerType type) {
                 if (testMethod.isAnnotationPresent(MethodOwner.class)) {
                     MethodOwner methodAnnotation = testMethod.getAnnotation(MethodOwner.class);
                     switch (type) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.251.139/sonarqube/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.251.139/sonarqube/coding_rules#rule_key=squid%3AS134)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.251.139/sonarqube/coding_rules#rule_key=squid%3AS1301)